packages/harmony/src/components/segmented-control/SegmentedControl.tsx

```diff
@@ -127,12 +127,12 @@ export const SegmentedControl = <T extends string>(
                 disabled={isOptionDisabled}
               />
               <Text
+                variant='body'
+                strength='strong'
+                css={(theme) => ({ lineHeight: theme.typography.lineHeight.s })}
                 color={
                   option.variant ?? (isOptionDisabled ? 'subdued' : 'default')
                 }
-                css={{
-                  cursor: isOptionDisabled ? 'auto' : 'pointer'
-                }}
```

Comment:
is this handles somewhere else? do we want to remove this?

Reply:
yeah it is handled on the <input> element